flutter/lib/src/event_processor/url_filter/html_url_filter_event_processor.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:html' as html show window, Window;
+
+import '../../../sentry_flutter.dart';
+import 'url_filter_event_processor.dart';
+// ignore: implementation_imports
+import 'package:sentry/src//utils/regex_utils.dart';
+
+// ignore_for_file: invalid_use_of_internal_member
+
+UrlFilterEventProcessor urlFilterEventProcessor(SentryFlutterOptions options) =>
+    WebUrlFilterEventProcessor(options);
+
+class WebUrlFilterEventProcessor implements UrlFilterEventProcessor {
+  WebUrlFilterEventProcessor(
+    this._options,
+  );
+
+  final html.Window _window = html.window;
+
+  final SentryFlutterOptions _options;
+
+  @override
+  SentryEvent? apply(SentryEvent event, Hint hint) {
+    final url = event.request?.url ?? _window.location.toString();
```

Comment:
I don't think `event.request?.url` is correct here since it is the http request url and not the current window url

we may have to use the stackframes to receive the url if we don't have other options

Reply:
yes this is true. I thought this is part of this feature, but we should keep this in mind if some problems or discussions are occurring.

Reply:
did you check if this is this not an issue?

Reply:
cc @martinhaintz

Reply:
the unit tests are all passing, so from this perspective it works. 
in the issue #626 it is not exactly defined, if only the current web application with a specific url or also the request which are sent to a specific url should be allowed/denied.
it looks like the javascript implementation distinguish between the application url which can be set in [`global options`](https://docs.sentry.io/platforms/javascript/configuration/options/#deny-urls) and scripts which are loaded from other sources in the [`inboundFilters`](https://docs.sentry.io/platforms/javascript/configuration/integrations/inboundfilters/#allowurls). 
It is not applying the rules based on request to other resources. But this can also be a handy feature.

Any suggestions how we should continue?

Reply:
> It is not applying the rules based on request to other resources

this was my understanding here as well, let's focus on current web application urls first and foremost. so that's why I think `event.request?.url` won't directly work since it also includes http request urls

Reply:
checking `_window.location.toString()`will not be sufficient?

Reply:
depends, let's take this example: denyUrl: example.com/deny

user triggers an error in example.com (which should be allowed) and then quickly navigates to example.com/deny but the event processing is not done yet and fetches `window.location` at a later time

just some thoughts, it might not be true and might be worth investigating, using `window.location` is definitely easier

but in any case we definitely know the correct path would be in the stackframes regardless of when the event would be processed

Reply:
also I think we'll have to check the absolute path of the stackframes, afaik that includes the url in flutter web